config/initializers/devise.rb

```diff
@@ -131,24 +131,24 @@
   # without confirming their account.
   # Default is 0.days, meaning the user cannot access the website without
   # confirming their account.
-  # config.allow_unconfirmed_access_for = 2.days
+   config.allow_unconfirmed_access_for = 2.days
```

Comment:
I'm a little dubious about this.  I think we should put confirmation emails in a separate PR because I'm worried that this iteration of them will create friction and make it harder for people to stay engaged.  I'm thinking we should restrict unconfirmed users in a more granular way, but we don't have the time to design that interaction this week.